### DISCUSSION TITLE
What's the difference between "Common" and "fleur"? I don't undertand this image comparison of Large V2 vs V3?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [openai/whisper#1773] [Q&A] [answered]
![image](https://github.com/openai/whisper/assets/148225825/b1e01e2f-ffd0-4def-9765-65b26fadec35)

**Comment by jongwook** [comment, marked as answer]
Hi, those are two datasets that we used for evaluating and comparing the model performance (and we didn't include the corresponding training splits in the training):

- Common Voice: https://commonvoice.mozilla.org/en/datasets
- Fleurs: https://huggingface.co/datasets/google/fleurs, https://arxiv.org/abs/2205.12446

**Comment by AIhasArrived** [comment]
Noob question, so for cantonese, large v3 is significantly better because he has a smaller WER?

**Reply by ryanheise** [reply]
Yes: Basically, smaller WER (Word Error Rate) means a smaller percentage of errors, and that's a reflection of it being better.
